src/main/java/org/broadinstitute/hellbender/utils/variant/writers/ReblockingGVCFBlockCombiner.java

```diff
@@ -260,7 +261,12 @@ public int getVcfOutputEnd() {
 
     public int getBufferEnd() { return bufferEnd; }
```

Comment:
Perhaps you must make sure that getBufferEnd won't ever return 0 or perhaps -1 if getBufferStart is 0 to be consistent..... for example could it happens that you return something like (0 .. 10M] when the buffer is empty? would that still make sense?  If you are using these methods in one place only (is that so, not sure) then perhaps you should instead have something different like ``` if (buffer.addIfApplies(vc)) { ``` that would return false it it didn't apply and thus it wasn't added. 
My point is to avoid using names that may result in misinterpretation due to the intuition based on the name.

Reply:
That would apply to the isHomRef on ./. calls above aswell.